go/ethadapter/interface.go

```diff
@@ -20,6 +20,7 @@ type EthClient interface {
 	SendTransaction(signedTx *types.Transaction) error               // issues an ethereum transaction (expects signed tx)
 	TransactionReceipt(hash gethcommon.Hash) (*types.Receipt, error) // fetches the ethereum transaction receipt
 	Nonce(address gethcommon.Address) (uint64, error)                // fetches the account nonce to use in the next transaction
+	GetBalance(address gethcommon.Address) (big.Int, error)          // fetches the balance of the account
```

Comment:
We need to add this method, as this is the client interface we use in the contract deployer, where we now need to check balances.